### PR TITLE
[Edit]: SQL Entry Typos

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

A couple of small typos in entries I previously submitted.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/data-types/data-types.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ SQL supports a range of data types across widely used classes of data, such as t
 - String or character types
 - Temporal types for dates and times
 
-These data types are found across all flavors of SQl; however, some versions of SQL may support several distinct data types of a particular class while others may only have one (the definitions below are specific to PostgreSQL).
+These data types are found across all flavors of SQL; however, some versions of SQL may support several distinct data types of a particular class while others may only have one (the definitions below are specific to PostgreSQL).
```
Comment (KTom101) [inline content/sql/concepts/data-types/data-types.md:23]: Can we also break this up into two sentences?

```suggestion
These data types are found across all flavors of SQL. However, some versions of SQL may support several distinct data types of a particular class while others may only have one (the definitions below are specific to PostgreSQL).
```

**Review by KTom101 — COMMENTED** [review]
@caupolicandiaz I mentioned two changes.

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz I mentioned two changes.

Thanks for the notes. All set.

**Review by KTom101 — APPROVED** [review]
@caupolicandiaz Looks good to go 👍

**Pull request merged**